tools/custom-action/prepare.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -eo pipefail
+
+export $(cat /.env | xargs) # Source the .env file
+cp -r /grammar ./wgsl/
+sudo python3 -m pip install --break-system-packages \
+  bikeshed
```

Comment:
also needs `--upgrade` I think.

Is sudo needed? these command should work as whatever user is running entrypoints (which may already be root anyway?)

Reply:
I had an error with non-sudo, retrying

Reply:
Non-sudo works with latest tweaks!